assets/scripts/lobby/sockets/sockets.js

```diff
@@ -271,7 +271,8 @@ socket.on('update-current-players-list', (currentPlayers) => {
   rankedPlayers.forEach((player) => {
     // if the current player exists, add it
     if (player) {
-      str = `<tr> <td>${player.displayUsername} ${generateBadgeString(player.badge)}</td> <td align="right">${player.ratingBadge} ${player.playerRating}</td> </tr>`;
+      str = `<tr> <td>${player.displayUsername} ${generateBadgeString(player.badge,)}</td> <td align="right">${player.ratingBadge} 
+      ${player.playerRating}</td> </tr>`;
```

Comment:
I don't think this is intentional. It could mess up the UI. Could you revert this change please?